integration/k3t/cluster.go

```diff
@@ -88,7 +88,7 @@ func NewCluster(ctx context.Context, t *testing.T, opts ...Option) Cluster {
 		},
 	}
 
-	clusterConfig, err := config.TransformSimpleToClusterConfig(ctx, runtimes.SelectedRuntime, simpleCfg)
+	clusterConfig, err := config.TransformSimpleToClusterConfig(ctx, runtimes.SelectedRuntime, simpleCfg, "")
```

Comment:
The argument that was added is a file name, which is only used if `simpleCfg` contained a non-empty Files field, but it's empty here.